apps/builder/src/features/analytics/components/StatsCards.tsx

```diff
@@ -10,6 +10,16 @@ import {
 import { Stats } from 'models'
 import React from 'react'
 
+const calculateCompletionRate = (
```

Comment:
```suggestion
const computeCompletionRate = (
```

I prefer this naming if it's ok with you! 👌